js/classes/entities/foreground/hud.js

```diff
@@ -22,6 +23,37 @@ class Hud {
         return 10;
     }
 
+    swapToCrosshair() {
+        let crosshairCursorUnclicked = 'url(../sprites/crosshair_unpressed.png) 16 16, auto';
+        let crosshairCursorClicked = 'url(../sprites/crosshair_pressed.png) 16 16, auto';
+
+        document.body.style.cursor = crosshairCursorUnclicked;
+
+        document.body.addEventListener('mousedown', () => {
+            document.body.style.cursor = crosshairCursorClicked;
+        });
+
+        document.body.addEventListener('mouseup', () => {
+            document.body.style.cursor = crosshairCursorUnclicked;
+        });
+    }
+
+    swapToPointer() {
+        let pointerCursorUnclicked = 'url(../sprites/pointer_unpressed.png), auto';
```

Comment:
If I recall from when you demoed this, the cursor was huge. Is auto a good size?